NEWS

```diff
@@ -430,8 +430,8 @@
       o New STORE structure and library to provide an interface to all
         sorts of data repositories.  Supports storage of public and
         private keys, certificates, CRLs, numbers and arbitrary blobs.
-	This library is unfortunately unfinished and unused withing
-	OpenSSL.
+	    This library is unfortunately unfinished and unused within
```

Comment:
This line seems to have had a tab instead of spaces.  Can you make sure there are no tabs?  (I currently see 3 lines starting with a tab, including the 2 you change here.)